.gitignore

```diff
@@ -59,6 +62,21 @@ $RECYCLE.BIN/
 src/target/
 src/.extract/
 
+# Test
+src/tomcat
+
 .idea
 src/rest-gateway.iml
 *.iml
+
+# build
+build/
+src/bin
```

Comment:
Lets rename the root level `src` directory to `adapter`, the current setup is a bit confusing when looking at the structure.